tasks/components/watchdog/install.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -e
+
+INTERVAL=$((CONFIG_DS_WATCHDOG_TIMEOUT/4))
```

Comment:
It looks like this removed the check for enable to run? https://github.com/embeddedTS/distro-seed/pull/31#discussion_r1309181496

If so, then would this script always run or would it only ever be run when DS_WATCHDOG is set?

Reply:
If a manifest.yaml has a config: option set, then it only runs any of its tasks if that config is set.  DS_WATCHDOG must be set for this script to run.